docs/typescript.mdx

```diff
@@ -39,10 +39,21 @@ To make the css prop work with pure TypeScript (without babel plugin) you need t
 
 ```tsx
 /** @jsx jsx */
-import { css, jsx } from '@emotion/core'
+import { jsx } from '@emotion/core'
+
+<div css={{ background: 'black' }} />
 ```
 
 As a result you may be not able to use react fragment shorthand syntax - `<></>`, but still you can use `<Fragment></Fragment>`.
+This is a limitation of the TypeScript compiler not being able to independently specify jsx pragma and jsxFrag pragma.
+
+You can still use the css helper and pass the className yourself (ensure you are importing from the `@emotion` package, not `@emotion/core`).
```

Comment:
I'm not sure if this should be included in this sections - seem a little bit off-putting, especially given that we don't mention this in non-TS sections (as far as i know). 

And also slight nit - `@emotion` should be `emotion`

Reply:
I think that this is a crucial bit of information and should be included both here - and in the non TS docs. "Off putting" up front is better than broken and surprising in hindsight.

Reply:
Not sure about that @Andarist, `@emotion/core` is mentioned everywhere else. I personally don't use the css prop so I pulled this info out of somewhere when I noticed it. It felt like something which should be in the docs